include/wx/webpdecoder.h

```diff
@@ -0,0 +1,57 @@
+/////////////////////////////////////////////////////////////////////////////
+// Name:        wx/webpdecoder.h
+// Purpose:     wxWebPDecoder, WebP Animation decoder
+// Author:
+// Copyright:
+// Licence:     wxWindows licence
+/////////////////////////////////////////////////////////////////////////////
+
+#ifndef _WX_WEBPDECODER_H
+#define _WX_WEBPDECODER_H
+
+#include "wx/defs.h"
+
+#if wxUSE_STREAMS && wxUSE_LIBWEBP
+
+#include "wx/stream.h"
+#include "wx/image.h"
+#include "wx/animdecod.h"
+#include "wx/imagwebp.h"
+
+
+class WXDLLIMPEXP_CORE wxWebPDecoder : public wxAnimationDecoder
+{
+public:
+    wxWebPDecoder();
+    ~wxWebPDecoder();
+
+    virtual wxSize GetFrameSize(unsigned int frame) const override;
+    virtual wxPoint GetFramePosition(unsigned int frame) const override;
+    virtual wxAnimationDisposal GetDisposalMethod(unsigned int frame) const override;
+    virtual long GetDelay(unsigned int frame) const override;
+    virtual wxColour GetTransparentColour(unsigned int frame) const override;
+
+    virtual bool Load(wxInputStream& stream) override;
+    bool ConvertToImage(unsigned int frame, wxImage* image) const override;
+
+    wxAnimationDecoder* Clone() const override
+    {
+        return new wxWebPDecoder;
+    }
+    wxAnimationType GetType() const override
+    {
+        return wxANIMATION_TYPE_WEBP;
+    }
```

Comment:
Just for consistency with the functions above:

```suggestion
    virtual bool ConvertToImage(unsigned int frame, wxImage* image) const override;

    virtual wxAnimationDecoder* Clone() const override
    {
        return new wxWebPDecoder;
    }
    virtual wxAnimationType GetType() const override
    {
        return wxANIMATION_TYPE_WEBP;
    }
```